api/src/services/listing-csv-export.service.ts

```diff
@@ -672,20 +694,27 @@ export class ListingCsvExporterService implements CsvExporterServiceInterface {
               .join(',');
           },
         },
-        {
-          path: 'listingMultiselectQuestions',
-          label: 'Housing Programs',
-          format: (val: ListingMultiselectQuestion[]): string => {
-            return val
-              .filter(
-                (question) =>
-                  question.multiselectQuestions.applicationSection ===
-                  'programs',
-              )
-              .map((question) => question.multiselectQuestions.text)
-              .join(',');
-          },
-        },
+        ...(doAnyJurisdictionHaveFeatureFlagSet(
+          user.jurisdictions,
+          FeatureFlagEnum.swapCommunityTypeWithPrograms,
```

Comment:
Issue: This is not ideal in that it hides the programs for all listings if only 1 jurisdiction has this flag set. Ideally for each listing we would look at this feature flag. But I don't think this CSV export file is setup to handle that case.

I'd recommend we do the inverse of this. Only hide the programs if all jurisdictions have that feature flag set. The same goes for the "community types". In production this solution should work as Detroit will only ever have 1 jurisdiction and all of our other jurisdictions favor not swapping these out.